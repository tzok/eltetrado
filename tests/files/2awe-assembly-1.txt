XXXX|1|A|U|1	cWH	XXXX|1|D|U|1	0
XXXX|1|A|G|3	cHW	XXXX|1|C|G|3	0
XXXX|1|A|G|3	cWH	XXXX|1|D|G|3	0
XXXX|1|A|G|5	cHW	XXXX|1|C|G|5	0
XXXX|1|A|G|5	cWH	XXXX|1|D|G|5	0
XXXX|1|A|U|6	cHW	XXXX|1|D|U|6	0
XXXX|1|A|U|6	cWH	XXXX|1|C|U|6	0
XXXX|1|B|U|1	cHW	XXXX|1|D|U|1	0
XXXX|1|B|U|1	cWH	XXXX|1|C|U|1	0
XXXX|1|B|G|3	cHW	XXXX|1|D|G|3	0
XXXX|1|B|G|3	cWH	XXXX|1|C|G|3	0
XXXX|1|B|G|5	cHW	XXXX|1|D|G|5	0
XXXX|1|B|G|5	cWH	XXXX|1|C|G|5	0
XXXX|1|B|U|6	cHW	XXXX|1|C|U|6	0
XXXX|1|B|U|6	cWH	XXXX|1|D|U|6	0
XXXX|1|C|U|1	cHW	XXXX|1|B|U|1	0
XXXX|1|C|G|3	cHW	XXXX|1|B|G|3	0
XXXX|1|C|G|3	cWH	XXXX|1|A|G|3	0
XXXX|1|C|G|5	cHW	XXXX|1|B|G|5	0
XXXX|1|C|G|5	cWH	XXXX|1|A|G|5	0
XXXX|1|C|U|6	cHW	XXXX|1|A|U|6	0
XXXX|1|C|U|6	cWH	XXXX|1|B|U|6	0
XXXX|1|D|U|1	cHW	XXXX|1|A|U|1	0
XXXX|1|D|U|1	cWH	XXXX|1|B|U|1	0
XXXX|1|D|G|3	cHW	XXXX|1|A|G|3	0
XXXX|1|D|G|3	cWH	XXXX|1|B|G|3	0
XXXX|1|D|G|5	cHW	XXXX|1|A|G|5	0
XXXX|1|D|G|5	cWH	XXXX|1|B|G|5	0
XXXX|1|D|U|6	cHW	XXXX|1|B|U|6	0
XXXX|1|D|U|6	cWH	XXXX|1|A|U|6	0
XXXX|1|E|U|1	cHW	XXXX|1|G|U|1	0
XXXX|1|E|U|1	cWH	XXXX|1|H|U|1	0
XXXX|1|E|G|3	cHW	XXXX|1|G|G|3	0
XXXX|1|E|G|3	cWH	XXXX|1|H|G|3	0
XXXX|1|E|G|5	cHW	XXXX|1|G|G|5	0
XXXX|1|E|G|5	cWH	XXXX|1|H|G|5	0
XXXX|1|E|U|6	cHW	XXXX|1|H|U|6	0
XXXX|1|E|U|6	cWH	XXXX|1|G|U|6	0
XXXX|1|F|U|1	cHW	XXXX|1|H|U|1	0
XXXX|1|F|G|3	cHW	XXXX|1|H|G|3	0
XXXX|1|F|G|3	cWH	XXXX|1|G|G|3	0
XXXX|1|F|G|5	cHW	XXXX|1|H|G|5	0
XXXX|1|F|G|5	cWH	XXXX|1|G|G|5	0
XXXX|1|F|U|6	cHW	XXXX|1|G|U|6	0
XXXX|1|F|U|6	cWH	XXXX|1|H|U|6	0
XXXX|1|G|U|1	cWH	XXXX|1|E|U|1	0
XXXX|1|G|G|3	cHW	XXXX|1|F|G|3	0
XXXX|1|G|G|3	cWH	XXXX|1|E|G|3	0
XXXX|1|G|G|5	cHW	XXXX|1|F|G|5	0
XXXX|1|G|G|5	cWH	XXXX|1|E|G|5	0
XXXX|1|G|U|6	cHW	XXXX|1|E|U|6	0
XXXX|1|G|U|6	cWH	XXXX|1|F|U|6	0
XXXX|1|H|U|1	cHW	XXXX|1|E|U|1	0
XXXX|1|H|U|1	cWH	XXXX|1|F|U|1	0
XXXX|1|H|G|3	cHW	XXXX|1|E|G|3	0
XXXX|1|H|G|3	cWH	XXXX|1|F|G|3	0
XXXX|1|H|G|5	cHW	XXXX|1|E|G|5	0
XXXX|1|H|G|5	cWH	XXXX|1|F|G|5	0
XXXX|1|H|U|6	cHW	XXXX|1|F|U|6	0
XXXX|1|H|U|6	cWH	XXXX|1|E|U|6	0
XXXX|1|A|U|1	ns33	XXXX|1|H|U|1	0
XXXX|1|A|U|1	ns53	XXXX|1|F|BGM|2	0
XXXX|1|A|U|1	s33	XXXX|1|F|U|1	0
XXXX|1|A|U|1	s53	XXXX|1|H|BGM|2	0
XXXX|1|A|BGM|2	s35	XXXX|1|F|U|1	0
XXXX|1|A|BGM|2	s35	XXXX|1|H|U|1	0
XXXX|1|A|BGM|2	s55	XXXX|1|A|G|3	0
XXXX|1|A|BGM|2	s55	XXXX|1|C|G|3	0
XXXX|1|A|G|3	s35	XXXX|1|A|G|5	0
XXXX|1|A|G|3	s35	XXXX|1|C|G|5	0
XXXX|1|A|G|3	s55	XXXX|1|A|BGM|2	0
XXXX|1|A|G|3	s55	XXXX|1|D|BGM|2	0
XXXX|1|A|G|5	s33	XXXX|1|A|U|6	0
XXXX|1|A|G|5	s33	XXXX|1|D|U|6	0
XXXX|1|A|G|5	s53	XXXX|1|A|G|3	0
XXXX|1|A|G|5	s53	XXXX|1|D|G|3	0
XXXX|1|A|U|6	s33	XXXX|1|A|G|5	0
XXXX|1|A|U|6	s33	XXXX|1|C|G|5	0
XXXX|1|B|U|1	ns33	XXXX|1|G|U|1	0
XXXX|1|B|U|1	s33	XXXX|1|E|U|1	0
XXXX|1|B|U|1	s53	XXXX|1|E|BGM|2	0
XXXX|1|B|U|1	s53	XXXX|1|G|BGM|2	0
XXXX|1|B|BGM|2	s35	XXXX|1|E|U|1	0
XXXX|1|B|BGM|2	s35	XXXX|1|G|U|1	0
XXXX|1|B|BGM|2	s55	XXXX|1|B|G|3	0
XXXX|1|B|BGM|2	s55	XXXX|1|D|G|3	0
XXXX|1|B|G|3	s35	XXXX|1|B|G|5	0
XXXX|1|B|G|3	s35	XXXX|1|D|G|5	0
XXXX|1|B|G|3	s55	XXXX|1|C|BGM|2	0
XXXX|1|B|G|3	s55	XXXX|1|B|BGM|2	0
XXXX|1|B|G|5	s33	XXXX|1|C|U|6	0
XXXX|1|B|G|5	s33	XXXX|1|B|U|6	0
XXXX|1|B|G|5	s53	XXXX|1|B|G|3	0
XXXX|1|B|G|5	s53	XXXX|1|C|G|3	0
XXXX|1|B|U|6	s33	XXXX|1|B|G|5	0
XXXX|1|B|U|6	s33	XXXX|1|D|G|5	0
XXXX|1|C|U|1	s33	XXXX|1|G|U|1	0
XXXX|1|C|U|1	s53	XXXX|1|G|BGM|2	0
XXXX|1|C|U|1	s53	XXXX|1|F|BGM|2	0
XXXX|1|C|BGM|2	ns35	XXXX|1|F|U|1	0
XXXX|1|C|BGM|2	ns35	XXXX|1|G|U|1	0
XXXX|1|C|BGM|2	s55	XXXX|1|B|G|3	0
XXXX|1|C|BGM|2	s55	XXXX|1|C|G|3	0
XXXX|1|C|G|3	s35	XXXX|1|C|G|5	0
XXXX|1|C|G|3	s35	XXXX|1|B|G|5	0
XXXX|1|C|G|3	s55	XXXX|1|A|BGM|2	0
XXXX|1|C|G|3	s55	XXXX|1|C|BGM|2	0
XXXX|1|C|G|5	s33	XXXX|1|A|U|6	0
XXXX|1|C|G|5	s33	XXXX|1|C|U|6	0
XXXX|1|C|G|5	s53	XXXX|1|A|G|3	0
XXXX|1|C|G|5	s53	XXXX|1|C|G|3	0
XXXX|1|C|U|6	s33	XXXX|1|B|G|5	0
XXXX|1|C|U|6	s33	XXXX|1|C|G|5	0
XXXX|1|D|U|1	ns33	XXXX|1|E|U|1	0
XXXX|1|D|U|1	s33	XXXX|1|H|U|1	0
XXXX|1|D|U|1	s53	XXXX|1|E|BGM|2	0
XXXX|1|D|U|1	s53	XXXX|1|H|BGM|2	0
XXXX|1|D|BGM|2	s35	XXXX|1|H|U|1	0
XXXX|1|D|BGM|2	s35	XXXX|1|E|U|1	0
XXXX|1|D|BGM|2	s55	XXXX|1|A|G|3	0
XXXX|1|D|BGM|2	s55	XXXX|1|D|G|3	0
XXXX|1|D|G|3	s35	XXXX|1|D|G|5	0
XXXX|1|D|G|3	s35	XXXX|1|A|G|5	0
XXXX|1|D|G|3	s55	XXXX|1|D|BGM|2	0
XXXX|1|D|G|3	s55	XXXX|1|B|BGM|2	0
XXXX|1|D|G|5	s33	XXXX|1|B|U|6	0
XXXX|1|D|G|5	s33	XXXX|1|D|U|6	0
XXXX|1|D|G|5	s53	XXXX|1|B|G|3	0
XXXX|1|D|G|5	s53	XXXX|1|D|G|3	0
XXXX|1|D|U|6	s33	XXXX|1|A|G|5	0
XXXX|1|D|U|6	s33	XXXX|1|D|G|5	0
XXXX|1|E|U|1	ns33	XXXX|1|D|U|1	0
XXXX|1|E|U|1	s33	XXXX|1|B|U|1	0
XXXX|1|E|U|1	s53	XXXX|1|B|BGM|2	0
XXXX|1|E|U|1	s53	XXXX|1|D|BGM|2	0
XXXX|1|E|BGM|2	s35	XXXX|1|B|U|1	0
XXXX|1|E|BGM|2	s35	XXXX|1|D|U|1	0
XXXX|1|E|BGM|2	s55	XXXX|1|E|G|3	0
XXXX|1|E|BGM|2	s55	XXXX|1|G|G|3	0
XXXX|1|E|G|3	s35	XXXX|1|E|G|5	0
XXXX|1|E|G|3	s35	XXXX|1|G|G|5	0
XXXX|1|E|G|3	s55	XXXX|1|E|BGM|2	0
XXXX|1|E|G|3	s55	XXXX|1|H|BGM|2	0
XXXX|1|E|G|5	s33	XXXX|1|H|U|6	0
XXXX|1|E|G|5	s33	XXXX|1|E|U|6	0
XXXX|1|E|G|5	s53	XXXX|1|E|G|3	0
XXXX|1|E|G|5	s53	XXXX|1|H|G|3	0
XXXX|1|E|U|6	s33	XXXX|1|E|G|5	0
XXXX|1|E|U|6	s33	XXXX|1|G|G|5	0
XXXX|1|F|U|1	ns53	XXXX|1|C|BGM|2	0
XXXX|1|F|U|1	s33	XXXX|1|A|U|1	0
XXXX|1|F|U|1	s53	XXXX|1|A|BGM|2	0
XXXX|1|F|BGM|2	ns35	XXXX|1|A|U|1	0
XXXX|1|F|BGM|2	s35	XXXX|1|C|U|1	0
XXXX|1|F|BGM|2	s55	XXXX|1|H|G|3	0
XXXX|1|F|BGM|2	s55	XXXX|1|F|G|3	0
XXXX|1|F|G|3	s35	XXXX|1|H|G|5	0
XXXX|1|F|G|3	s35	XXXX|1|F|G|5	0
XXXX|1|F|G|3	s55	XXXX|1|G|BGM|2	0
XXXX|1|F|G|3	s55	XXXX|1|F|BGM|2	0
XXXX|1|F|G|5	s33	XXXX|1|F|U|6	0
XXXX|1|F|G|5	s33	XXXX|1|G|U|6	0
XXXX|1|F|G|5	s53	XXXX|1|G|G|3	0
XXXX|1|F|G|5	s53	XXXX|1|F|G|3	0
XXXX|1|F|U|6	s33	XXXX|1|F|G|5	0
XXXX|1|F|U|6	s33	XXXX|1|H|G|5	0
XXXX|1|G|U|1	ns33	XXXX|1|B|U|1	0
XXXX|1|G|U|1	ns53	XXXX|1|C|BGM|2	0
XXXX|1|G|U|1	s33	XXXX|1|C|U|1	0
XXXX|1|G|U|1	s53	XXXX|1|B|BGM|2	0
XXXX|1|G|BGM|2	s35	XXXX|1|C|U|1	0
XXXX|1|G|BGM|2	s35	XXXX|1|B|U|1	0
XXXX|1|G|BGM|2	s55	XXXX|1|F|G|3	0
XXXX|1|G|BGM|2	s55	XXXX|1|G|G|3	0
XXXX|1|G|G|3	s35	XXXX|1|F|G|5	0
XXXX|1|G|G|3	s35	XXXX|1|G|G|5	0
XXXX|1|G|G|3	s55	XXXX|1|E|BGM|2	0
XXXX|1|G|G|3	s55	XXXX|1|G|BGM|2	0
XXXX|1|G|G|5	s33	XXXX|1|E|U|6	0
XXXX|1|G|G|5	s33	XXXX|1|G|U|6	0
XXXX|1|G|G|5	s53	XXXX|1|E|G|3	0
XXXX|1|G|G|5	s53	XXXX|1|G|G|3	0
XXXX|1|G|U|6	s33	XXXX|1|F|G|5	0
XXXX|1|G|U|6	s33	XXXX|1|G|G|5	0
XXXX|1|H|U|1	ns33	XXXX|1|A|U|1	0
XXXX|1|H|U|1	s33	XXXX|1|D|U|1	0
XXXX|1|H|U|1	s53	XXXX|1|A|BGM|2	0
XXXX|1|H|U|1	s53	XXXX|1|D|BGM|2	0
XXXX|1|H|BGM|2	s35	XXXX|1|A|U|1	0
XXXX|1|H|BGM|2	s35	XXXX|1|D|U|1	0
XXXX|1|H|BGM|2	s55	XXXX|1|H|G|3	0
XXXX|1|H|BGM|2	s55	XXXX|1|E|G|3	0
XXXX|1|H|G|3	s35	XXXX|1|H|G|5	0
XXXX|1|H|G|3	s35	XXXX|1|E|G|5	0
XXXX|1|H|G|3	s55	XXXX|1|H|BGM|2	0
XXXX|1|H|G|3	s55	XXXX|1|F|BGM|2	0
XXXX|1|H|G|5	s33	XXXX|1|H|U|6	0
XXXX|1|H|G|5	s33	XXXX|1|F|U|6	0
XXXX|1|H|G|5	s53	XXXX|1|H|G|3	0
XXXX|1|H|G|5	s53	XXXX|1|F|G|3	0
XXXX|1|H|U|6	s33	XXXX|1|E|G|5	0
XXXX|1|H|U|6	s33	XXXX|1|H|G|5	0
