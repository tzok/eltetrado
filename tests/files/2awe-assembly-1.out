Chain order: A C B D E H F G
n4-helix with 4 tetrads
  Op* VIII n/a quadruplex with 4 tetrads
    A.BGM2 C.BGM2 B.BGM2 D.BGM2 cWH cWH cWH cWH O+ VIIIb planarity=0.34 ions_channel=K,K 
      direction=parallel rise=3.57 twist=14.63
    A.G3 C.G3 B.G3 D.G3 cHW cHW cHW cHW O- VIIIa planarity=0.68 ions_channel=K 
      direction=parallel rise=3.38 twist=53.44
    A.G5 C.G5 B.G5 D.G5 cHW cHW cHW cHW O- VIIIa planarity=0.75 ions_channel=K 
      direction=parallel rise=3.46 twist=38.55
    A.U6 C.U6 B.U6 D.U6 cWH cWH cWH cWH O+ VIIIa planarity=1.15  ions_outside=A.U6: [SR] C.U6: [K]

    Tracts:
      A.BGM2, A.G3, A.G5, A.U6
      C.BGM2, C.G3, C.G5, C.U6
      B.BGM2, B.G3, B.G5, B.U6
      D.BGM2, D.G3, D.G5, D.U6

    Bulges:
      A.U4, C.U4, B.U4, D.U4

n4-helix with 4 tetrads
  Op* VIII n/a quadruplex with 4 tetrads
    E.BGM2 G.BGM2 F.BGM2 H.BGM2 cWH cWH cWH cWH O- VIIIb planarity=0.29 ions_channel=K,K 
      direction=parallel rise=3.54 twist=13.64
    E.G3 G.G3 F.G3 H.G3 cHW cHW cHW cHW O+ VIIIa planarity=0.67 ions_channel=K 
      direction=parallel rise=3.39 twist=53.15
    E.G5 G.G5 F.G5 H.G5 cHW cHW cHW cHW O+ VIIIa planarity=0.81 ions_channel=K 
      direction=parallel rise=3.47 twist=39.69
    E.U6 G.U6 F.U6 H.U6 cWH cWH cWH cWH O- VIIIa planarity=1.21  ions_outside=F.U6: [K] G.U6: [K]

    Tracts:
      E.BGM2, E.G3, E.G5, E.U6
      G.BGM2, G.G3, G.G5, G.U6
      F.BGM2, F.G3, F.G5, F.U6
      H.BGM2, H.G3, H.G5, H.U6

    Bulges:
      E.U4, F.U4, H.U4

UGGUG-UUGGUG-UUGGUG-UUGGUG-UUGGUG-UUGGUG-UUGGUG-UUGGGU
.([.{-[.)(.{-].[).}-(.]].}-).((.[-{.{).]-(.}[.{-).)]}}
.((.[-{.{).]-(.}[.{-).)].}-}.([.{-[.)(.{-].[).}-(.]]})
